types/react-faux-dom/index.d.ts

```diff
@@ -0,0 +1,29 @@
+// Type definitions for react-faux-dom 3.1
+// Project: https://github.com/Olical/react-faux-dom
+// Definitions by: Ali Taheri Moghaddar <https://github.com/alitaheri>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as React from "react";
+
+export class Element extends HTMLElement {
+    style: any;
+
+    constructor(nodeName: string, parentNode?: Element);
+
+    toReact(): React.ReactElement<any>;
+}
+
+export const defaultView: {
+    getComputedStyle(node: Element): { getPropertyValue(name: string): string };
+};
+
+export namespace mixins {
+    const core: any;
```

Comment:
Any way to type this? I'm not familiar with the react typings but it looks like they use `Mixin<P, S>` for the type of a mixin.